nibabel/freesurfer/io.py

```diff
@@ -129,6 +170,14 @@ def write_geometry(filepath, coords, faces, create_stamp=None):
         coords.astype('>f4').reshape(-1).tofile(fobj)
         faces.astype('>i4').reshape(-1).tofile(fobj)
 
+        # Add volume info, if given
+        if volume_info is not None and len(volume_info) > 0:
+            for key, val in volume_info.items():
+                if key == 'head':
+                    val.tofile(fobj)
+                    continue
+                fobj.write('{0}={1}'.format(key, val).encode('utf-8'))
```

Comment:
Input has spaces around `=`, so to replicate the original file as closely as possible, I'd use `'{0} = {1}'`.